x-pack/plugins/snapshot_restore/public/application/app.tsx

```diff
@@ -87,27 +86,25 @@ export const App: React.FunctionComponent = () => {
             </Routes>
           </div>
         ) : (
-          <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
-            <NotAuthorizedSection
-              title={
-                <FormattedMessage
-                  id="xpack.snapshotRestore.app.deniedPrivilegeTitle"
-                  defaultMessage="You're missing cluster privileges"
-                />
-              }
-              message={
-                <FormattedMessage
-                  id="xpack.snapshotRestore.app.deniedPrivilegeDescription"
-                  defaultMessage="To use Snapshot and Restore, you must have {privilegesCount,
-                    plural, one {this cluster privilege} other {these cluster privileges}}: {missingPrivileges}."
-                  values={{
-                    missingPrivileges: privilegesMissing.cluster!.join(', '),
-                    privilegesCount: privilegesMissing.cluster!.length,
-                  }}
-                />
-              }
-            />
-          </EuiPageContent>
+          <NotAuthorizedSection
```

Comment:
Note: This is fine for now. However, it will not render as expected until the `NotAuthorizedSection` component in `es_ui_shared` is updated to use `EuiPageTemplate.EmptyPrompt`. [Related code](https://github.com/elastic/kibana/blob/main/src/plugins/es_ui_shared/__packages_do_not_import__/authorization/components/not_authorized_section.tsx#L18).

This should be addressed via https://github.com/elastic/kibana/issues/163018